### PR TITLE
Background info

### PR DESCRIPTION
**Pull request by michaelrkn**
Addresses #79.

### CODE REVIEW

File: source/ember-cli/other-terms.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+## Shadow DOM
+**Not to be confused with Virtual DOM**. Ember's Shadow DOM is still a work in progress, but is basically a proposed way to have an "isolated" DOM encapsulated within your app's DOM.
+
+Creating a re-usable "widget" or control might be a good use-case for this. Browsers implement some of their controls using their own version of a shadow DOM.
+
+Here's a good [explanation of Shadown DOM](http://glazkov.com/2011/01/14/what-the-heck-is-shadow-dom/), as well as the [W3C Working Draft](http://www.w3.org/TR/shadow-dom/).
+
+
+## Virtual DOM
+**Not to be confused with Shadow DOM**. Ember's [Glimmer enginer](https://github.com/emberjs/ember.js/pull/10501) will implement a Virtual DOM to dramatically improve rendering performance.
```
Comment (bsclifton) [inline source/ember-cli/other-terms.md]: s/enginer/engine :)

File: source/ember-cli/other-terms.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+## Transpile
+Transpile means to convert from one language to another of a similar abstraction level. In Ember, you'll run into transpiling most often in the context of transpiling your ES6 syntax JavaScript to JavaScript that is supported by current browsers. Ember transpiles ES6 using [Babel](https://babeljs.io/) via the [ember-cli-babel](https://github.com/babel/ember-cli-babel) plugin.
+
+
+## Shadow DOM
+**Not to be confused with Virtual DOM**. Ember's Shadow DOM is still a work in progress, but is basically a proposed way to have an "isolated" DOM encapsulated within your app's DOM.
+
+Creating a re-usable "widget" or control might be a good use-case for this. Browsers implement some of their controls using their own version of a shadow DOM.
+
+Here's a good [explanation of Shadown DOM](http://glazkov.com/2011/01/14/what-the-heck-is-shadow-dom/), as well as the [W3C Working Draft](http://www.w3.org/TR/shadow-dom/).
```
Comment (bsclifton) [inline source/ember-cli/other-terms.md]: s/Shadown/Shadow

File: data/guides.yml [hunk 3]
```diff
@@ -10,8 +10,10 @@
   chapters:
     - title: "Installing Ember"
       url: "index"
-    - title: "Glossary"
-      url: "glossary"
+    - title: "Background Concepts"
```
Comment (joostdevries) [inline data/guides.yml]: I'd use "Foundation" or "Key background concepts" here. If you're writing this for new devs, they **must** grasp these or they'll have a bad time.

File: source/ember-cli/background.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+Before you get started building your first Ember app, there are a few concepts
+you should be familiar with:
```
Comment (joostdevries) [inline source/ember-cli/background.md]: I'd write one/two sentences per item here.

**Comment by joostdevries** [comment]
I like the fact that you're trying to separate the "must understands" from the "would be good to understand/you should already understand" but I'd name it as such. If you don't get what's listed in `background`, you're going to have a bad time. 

Another thing: we should have a clear idea on the different problems we want to solve:
1. Have people understand the foundation before getting started
2. Teach people the right conventions
3. Help people troubleshoot issues arising from not doing 1&2.

Right now, this seems to focus on 1. I've already commented in other issues that for example teaching people to `import` is highly critical or they will end up with dozens of JSHint errors.

**Comment by joostdevries** [comment]
Just read through the rest of the pages and I agree with @trek that it would make sense to move this to its own section. There's actually a dedicated `Concepts` section in the Guide which now has `Core concepts` and `Naming conventions`. The `Background info` and `Glossary` sections would make most sense there I guess.

**Comment by michaelrkn** [comment]
Thanks for the feedback. I've updated the PR. Let me know what you all think now.

**Comment by joostdevries** [comment]
@michaelrkn put my suggestions in #89 . (did not see your comment)

**Comment by michaelrkn** [comment]
Is there a reason this hasn't been merged? I just had to go back and rebase to fix merge conflicts with other changes since I opened this.

/cc @trek @rwjblue

**Comment by michaelrkn** [comment]
Closing this in favor of a simpler PR with fewer changes to make it easier to review

**Pull request closed without merging**